### PR TITLE
Updated naming convention of new fd

### PR DESCRIPTION
**Pull request by nasir-rahman9**
# Pull Request Template

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] My code follows the style guidelines of this project
- [ ] My code changes do not include any hardcoded secrets or passwords
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My Tech Lead is aware of any infrastructure changes that will cost money

### CODE REVIEW

File: app/modules/front-door/locals.tf [hunk 1]
```diff
@@ -1,8 +1,11 @@
 locals {
+  service_name = "back-office-applications-docs"
+  environment  = var.environment
```
Comment (beejjacobs) [inline app/modules/front-door/locals.tf:3]: Do we need a copy of environment as local?

File: app/modules/front-door/locals.tf [hunk 2]
```diff
@@ -1,8 +1,11 @@
 locals {
+  service_name = "back-office-applications-docs"
```
Comment (beejjacobs) [inline app/modules/front-door/locals.tf:2]: Is this meant to be at the module level? I'd expect this just in the stack maybe.

**Review by beejjacobs — CHANGES_REQUESTED** [review]
I think there's changes in the module that were perhaps intended for the stack?

File: app/modules/front-door/locals.tf (hunk 1 above)
Comment (nasir-rahman9) [reply app/modules/front-door/locals.tf]: removed from locals on new commit

**Review by beejjacobs — APPROVED** [review]

**Pull request merged**